setup-wizard/src/app/pages/success/success.page.html

```diff
@@ -76,6 +76,10 @@ <h2>LAN Instructions (Slightly Advanced):</h2>
           <a href="https://docs.start9.com/user-manual/general/lan-setup.html" target="_blank" rel="noreferrer"><b>here</b></a>.
         </p>
 
+	<p>
+	<b>Please note, that unless you reset this device, the embassy.local address will no longer connect to your Embassy from here on out.</b>
+	</p>	 
+
```

Comment:
This messaging doesnt seem necessary as the embassy.local is only useful if your embassy is not yet available

Reply:
@DrewAns Right, I just was temporarily thrown off by embassy.local not being available after successful initialization, and I was wondering if someone else may not find it as clear that embassy.local is down and that they need to use the new LAN address.

Reply:
@DrewAns Basically just wanting to make the new situation after initialization as obvious as possible for any user.  I can remove if everyone's final decision is to take it out, just my 2c as to why I thought it would be appropriate in there.

Reply:
I think it's useful

Reply:
Could maybe clarify what you mean by "reset this device", sounds like you mean a reboot

Reply:
Also the comma should go after `that`, not before :p

Reply:
My opinion is that this page is getting a bit cluttered, perhaps this belongs in the docs instead?

Reply:
> Could maybe clarify what you mean by "reset this device", sounds like you mean a reboot

@chrisguida  I thought reset meant reverting back to an initial start, and restart == reboot. What would a better way to word this be maybe? Since that word can be ambiguous.

@kn0wmad While I would agree this could have a home in the docs, I think it would be useful on this page since the information about the new situation would be presented immediately to the user, (it may not be immediate to a non-technical user to dig through docs upon a failure to go to embassy.local.) Speaking from personal experience I thought something had broken with my LAN connection until I thought to check the LAN section of the page.

Reply:
Up to you @kn0wmad

Reply:
I'm leaving this decision to @DrewAns; documentation is my territory, and I can definitely add it there.  Drew has final say on this page

Reply:
Valid
please change to
Please note, once setup is complete, the embassy.local address will no longer connect to your Embassy.

Reply:
@DrewAns  updated!